charter.html

```diff
@@ -165,13 +165,18 @@ <h2>Scope</h2>
             or cross context recognition</a> of users or
             enabling <a href="https://w3ctag.github.io/privacy-principles/#hl-recognition-same-site">same-site
             or same-context recognition</a> of users across the clearing of
-            state.
+            state
         </p>
         <p>
           The Working Group may consider designs that allow user agents for the
           same user — including non-browser agents, like Operating Systems — to
           collaborate in providing advertising features.
           </p>
+        <p>
+          We will use the 
+          <a href="https://w3ctag.github.io/privacy-principles/#boring-terminology">definitions established by the W3C and TAG</a> 
+          exclusively when discussing features. If a term has multiple definitions and one of those is established by existing W3C 
+          documentation, the W3C documentation always provides the correct definition. 
         </p>
```

Comment:
This is a little weird.

Charters exist to constrain activities.  Usually that is a good thing as it helps participants understand scope, but this seems overly constraining.  I would stick with the first sentence only.  Then the WG is able to debate whether a digression from established terminology is appropriate in a specific context, though they would be strongly discouraged from doing so.  This text is a very strong constraint that might prevent the WG from even considering this.

If this is important, then I'm not seeing a reason to go this way.